R/create_single_dose_dataset.R

```diff
@@ -550,15 +548,11 @@ create_single_dose_dataset <- function(dataset,
   if (nrow(value_check) > 0) {
     values_not_found <- paste0(value_check %>% select(!!dose_freq), collapse = ", ")
```

Comment:
current message has `\` between values:  `c(\"Q2D\", \"Q3D\", \"EVERY 2 WEEKS\")`
```r
Error in `create_single_dose_dataset()`:
! The following values of "EXDOSFRQ" in `dataset` do not appear in `lookup_table`: "c(\"Q2D\", \"Q3D\", \"EVERY 2 WEEKS\")"
```

```suggestion
    values_not_found <- value_check %>% select(!!dose_freq)
```